bundle/manifests/tempo-operator.clusterserviceversion.yaml

```diff
@@ -14,6 +14,7 @@ metadata:
         }
       ]
     capabilities: Basic Install
+    createdAt: "2023-03-01T18:24:59Z"
```

Comment:
oh - i assume we need something like this:
https://github.com/jaegertracing/jaeger-operator/blob/7e668d84b948b8366b46eaf5dfe0c0a849e943e4/hack/ignore-createdAt-bundle.sh

Reply:
Yes